cpp/src/file/tsfile_io_writer.cc

```diff
@@ -83,19 +83,35 @@ int TsFileIOWriter::start_file() {
     return ret;
 }
 
-int TsFileIOWriter::start_flush_chunk_group(const std::string &device_name) {
+int TsFileIOWriter::start_flush_chunk_group(const std::string &device_name,
+                                            bool is_aligned) {
     int ret = E_OK;
     if (RET_FAIL(write_byte(CHUNK_GROUP_HEADER_MARKER))) {
     } else if (RET_FAIL(write_string(device_name))) {
```

Comment:
Maybe it would be better to just add return statements here. The reader must check carefully to know if the error code is immediately returned.

Reply:
Removed `RET_FAIL`.